pkg/volume/util/volumepathhandler/volume_path_handler.go

```diff
@@ -97,11 +97,10 @@ func (v VolumePathHandler) MapDevice(devicePath string, mapPath string, linkName
 	// Check and create mapPath
 	_, err := os.Stat(mapPath)
 	if err != nil && !os.IsNotExist(err) {
-		klog.Errorf("cannot validate map path: %s", mapPath)
-		return err
+		return fmt.Errorf("Cannot validate map path: %s: %v", mapPath, err)
```

Comment:
errors start with lowercase (`"cannot validate..."`), so they can be easily chained with `":"`

Mockup of such chained error:
"pod startup failed: Cannot validate map path: permission denied".

This applies to whole PR.

Reply:
Fixed similar issues in this PR.